docs/how/delete-metadata.md

```diff
@@ -43,6 +43,9 @@ datahub delete --platform snowflake
 # Filters can be combined, which will select entities that match all filters.
 datahub delete --platform looker --entity-type chart
 datahub delete --platform bigquery --env PROD
+
+# You can also do recursive deletes for some entity types.
```

Comment:
Can we just specify those entities types, i.e. container and dataPlatformInstance